Xamarin.Forms.Platform.WP8/ButtonRenderer.cs

```diff
@@ -127,26 +137,31 @@ void UpdateContent()
 				case Button.ButtonContentLayout.ImagePosition.Top:
 					container.Orientation = Orientation.Vertical;
 					image.Margin = new WThickness(0, 0, 0, spacing);
+					container.Children.Add(image);
+					container.Children.Add(textBlock);
 					break;
 				case Button.ButtonContentLayout.ImagePosition.Bottom:
 					container.Orientation = Orientation.Vertical;
 					image.Margin = new WThickness(0, spacing, 0, 0);
+					container.Children.Add(textBlock);
+					container.Children.Add(image);
```

Comment:
Should we move all the common stuff out of the switch statement and cut down on the code duplication?